### PR TITLE
prompt for az vmware private-cloud delete

### PR DESCRIPTION
**Pull request by ctaggart**
This is for internal work item [9019400](https://msazure.visualstudio.com/One/_workitems/edit/9019400/). It prompts the user with `This will delete private cloud. Are you sure? (y/N)` when they try to delete a cloud. This is a breaking change and will need to be communicated appropriately. Any recommendation on how to communicate this breaking change? There will be another similar pull request soon.

For example:
```
(.venv) PS C:\Users\cataggar\io\azure-cli-extensions> az vmware private-cloud delete -g taggac-2020-12 -n taggac-pc
This will delete private cloud. Are you sure? (y/N):
(.venv) PS C:\Users\cataggar\io\azure-cli-extensions> az vmware private-cloud delete -g taggac-2020-12 -n taggac-pc
This will delete private cloud. Are you sure? (y/N): asfd
This will delete private cloud. Are you sure? (y/N): jnf
This will delete private cloud. Are you sure? (y/N): asdf
This will delete private cloud. Are you sure? (y/N): N
(.venv) PS C:\Users\cataggar\io\azure-cli-extensions> az vmware private-cloud delete -g taggac-2020-12 -n taggac-pc --yes
(.venv) PS C:\Users\cataggar\io\azure-cli-extensions>
```

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
vmware

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
hi @qwordy could you pls review?

**Comment by qwordy** [comment]
@ctaggart Make sure add it to release history and highlight it as BREAKING CHANGE. It is the convention in Azure CLI.

**Review by qwordy — APPROVED** [review]

**Comment by ctaggart** [comment]
Okay. I'll look into adding it to release history tomorrow.

**Comment by ctaggart** [comment]
I've updated the changelog and highlighted it as a breaking change.

File: src/vmware/CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,12 @@
 
 ## 2.0.0 (TBD)
```
Comment (qwordy) [inline src/vmware/CHANGELOG.md:3]: Do we replace TBD with a specific date before merging this PR?

Comment (ctaggart) [reply src/vmware/CHANGELOG.md:3]: There are more features coming. 2.0 is not ready just yet. This PR is. Would you prefer something else there?

Comment (qwordy) [reply src/vmware/CHANGELOG.md:3]: No problem. Do you want me to merge it?

Comment (ctaggart) [reply src/vmware/CHANGELOG.md:3]: Please do. I do not have access.

**Pull request merged**